bin/qcodes_logger_server

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import logging
+import zmq
+
+
+def broker(frontend_addres="tcp://*:5559", backend_address="tcp://*:5560"):
+    """
+    Simple XPUB/XSUB broker.
+    Listens for messages on the frontend and transparently pushes them to a
+    backend.
+    This allows to have centralized logging, from multiple processes
+    and to multiple consumers.
+    Messages sent but never forward (f.e.x if there aren't subscribers)
+    are quietly dropped.
+
+    Args:
+        frontend_addres (str): Interface to which the frontend is bound
+        backend_address (str): Interface to which the backend is bound
+
+    """
+    context = zmq.Context()
+    # Socket facing clients
+    frontend = context.socket(zmq.XSUB)
+    try:
+        frontend.bind(frontend_addres)
+        logging.info("XSUB listening at {}".format(frontend_addres))
+    except zmq.error.ZMQError:
+        logging.debug("Exiting. Broker is already running")
```

Comment:
well, coming back to this. It's going to work. Introducing real service discovery would be way to tricky given how dynamics things are.

If the ports are too standard or things breaks, we will fix as we go.